hugegraph-llm/src/hugegraph_llm/demo/rag_demo/app.py

```diff
@@ -105,6 +110,41 @@ def init_rag_ui() -> gr.Interface:
         with gr.Tab(label="5. Admin Tools 🛠"):
             create_admin_block()
 
+            def switch_language(current_lang):
+                new_lang = "CN" if current_lang == "EN" else "EN"
+                if new_lang == "CN":
+                    updates = (
+                        gr.update(value=f"prompt: {new_lang}"),
+                        gr.update(value=prompt.doc_input_text_CN),
+                        gr.update(value=prompt.extract_graph_prompt_CN),
+                        gr.update(value=prompt.answer_prompt_CN),
+                        gr.update(value=prompt.keywords_extract_prompt_CN),
+                        gr.update(value=prompt.gremlin_generate_prompt_CN),
+                    )
+                else:
+                    updates = (
+                        gr.update(value=f"prompt: {new_lang}"),
+                        gr.update(value=prompt.doc_input_text),
+                        gr.update(value=prompt.extract_graph_prompt),
+                        gr.update(value=prompt.answer_prompt),
+                        gr.update(value=prompt.keywords_extract_prompt),
+                        gr.update(value=prompt.gremlin_generate_prompt),
+                    )
```

Comment:
Note the consistency of prompts on the **API** and **UI** to avoid reading inconsistent prompts when the API accesses them